x-pack/plugins/cases/public/components/all_cases/all_cases_list.tsx

```diff
@@ -224,7 +210,8 @@ export const AllCasesList = React.memo<AllCasesListProps>(
     const tableRowProps = useCallback(
       (theCase: Case) => ({
         'data-test-subj': `cases-table-row-${theCase.id}`,
-        className: classnames({ isDisabled: theCase.type === CaseType.collection }),
+        // TODO: should we just remove this line?
+        className: classnames({ isDisabled: false }),
```

Comment:
@cnasikas do we still need the `isDisabled` prop? We only disabled the rows in the case list page if it was a parent to a sub case.

Reply:
Nope, you can delete the whole `className` property.